src/List/__tests__/List-test.js

```diff
@@ -0,0 +1,13 @@
+/* eslint-env mocha */
```

Comment:
this test can be removed. The list is just a component from the basicClassCreator, which already have tests.